images/macos/provision/configuration/CreateCleanAnkaTemplate.ps1

```diff
@@ -0,0 +1,232 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [Version] $MacOSVersion,
+    [ValidateNotNullOrEmpty()]
+    [String] $TemplateUsername,
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [String] $TemplatePassword,
+    [ValidateNotNullOrEmpty()]
+    [String] $RegistryUrl,
+    [Bool] $BetaSearch = $false,
+    [Int] $CpuCount = 6,
+    [Int] $RamSizeGb = 7,
+    [Int] $DiskSizeGb = 300,
+    [String] $DisplayResolution = "1920x1080"
+)
+
+$ErrorActionPreference = "Stop"
+
+function Get-MacOSInstaller {
+    param (
+        [Version] $MacOSVersion,
+        [Bool] $BetaSearch = $false
+    )
+
+    # Enroll machine to DeveloperSeed if we need beta and unenroll otherwise
+    if ($BetaSearch)
+    {
+        Write-Host "Beta Version requested. Enrolling machine to DeveloperSeed"
+        sudo /System/Library/PrivateFrameworks/Seeding.framework/Versions/Current/Resources/seedutil enroll DeveloperSeed | Out-Null
```

Comment:
nitpick: move `/System/Library/PrivateFrameworks/Seeding.framework/Versions/Current/Resources/seedutil` path to the separate variable before `if` to avoid duplication